samples/subsys/ipc/ipm_imx/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+    description: TBD
```

Comment:
Can we fill this information in?

Reply:
I have added the name and description, thank you.